db/1.10/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -78,6 +79,60 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
+== Understand Select and Stored Procedure Operations Warning Exceptions
+
+When the Database Connector *Select* and *Stored procedure* operations execute correctly but results are not present, Mule runtime engine (Mule) returns a `WARN` exception message instead of throwing the exception.
+
+This occurs when executing an invalid SQL statement or expecting that an error within a set of statements will collapse the procedure. In fact, JDBC returns a series of `ResultSets`, `UpdateCounts`, and `Exceptions`.
```

Comment:
I'm sure this is right, but I don't know what "expecting that an error within a set of statements will collapse the procedure" means.